pkg/clientcert/certificate.go

```diff
@@ -80,16 +80,18 @@ func IsCertificateValid(certData []byte, subject *pkix.Name) (bool, error) {
 	}
 
 	// check subject of certificates
+	organizationSets := sets.NewString(subject.Organization...)
+	organizationalUnitSets := sets.NewString(subject.OrganizationalUnit...)
 	for _, cert := range certs {
 		if cert.Subject.CommonName != subject.CommonName {
 			continue
 		}
 
-		if !reflect.DeepEqual(cert.Subject.Organization, subject.Organization) {
+		if !organizationSets.Equal(sets.NewString(cert.Subject.Organization...)) {
```

Comment:
i would want removing the comparison of o/ou now. We can add it when we think it is necessary.

Reply:
Removed.